components/ScrollUpButton/index.jsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from 'react';
+import { Button } from '@catho/quantum';
+import styles from './ScrollUpButton.module.css';
+
+const ScrollUpButton = () => {
+  const [scrollUpButton, setScrollUpButton] = useState(false);
+  useEffect(() => {
+    window.addEventListener('scroll', () => {
+      if (window.scrollY > 50) {
+        setScrollUpButton(true);
+      } else {
+        setScrollUpButton(false);
+      }
+    });
+  }, []);
+
+  const scrollTop = () => {
+    window.scrollTo({
+      top: 0,
+      behavior: 'auto',
+    });
+  };
+
+  return (
+    <>
+      <div className={styles.ScrollUpButton}>
+        {scrollUpButton && (
+          <Button size="small" onClick={scrollTop}>
+            Voltar ao Topo
+          </Button>
+        )}
+      </div>
+    </>
```

Comment:
```suggestion
      <div className={styles.ScrollUpButton}>
        {scrollUpButton && (
          <Button size="small" onClick={scrollTop}>
            Voltar ao Topo
          </Button>
        )}
      </div>
```
The use of fragment is recommended when there is more than one descendant. In this case, as there is only one div, you can remove it, keeping only the div